meta-facebook/minerva-ag/src/shell/sensor_threshold_shell.c

```diff
@@ -95,14 +97,21 @@ void cmd_get_sensor_threshold(const struct shell *shell, size_t argc, char **arg
 		     i++) {
 			result = check_supported_threshold_with_sensor_id(i);
 			if (result == 0) {
+				char sensor_name[MAX_AUX_SENSOR_NAME_LEN] = { 0 };
+
 				get_pdr_table_critical_high_and_low_with_sensor_id(
 					i, &critical_high, &critical_low);
+
+				pldm_get_sensor_name_via_sensor_id(i, sensor_name,
+								   sizeof(sensor_name));
+
 				shell_print(
 					shell,
-					"sensor ID: 0x%x  |  critical high: %10.3f  |  critical low: %10.3f",
-					i, critical_high, critical_low);
+					"sensor ID: 0x%x  |  sensor name: %-40s  |  critical high: %10.3f  |  critical low: %10.3f",
+					i, sensor_name, critical_high, critical_low);
 				critical_high = 0;
 				critical_low = 0;
+				memset(sensor_name, 0, sizeof(sensor_name));
```

Comment:
sensor_name is going to go out of scope outside if condition starting at line#99, I don't think you need to memset it 0 as its not being used again?

Reply:
I have fixed this issue, thank you.